src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -139,150 +172,180 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
-                  )}
-              </QuestionLabel>
-              <TextArea
-                data-testid="text"
-                error={Boolean(state.errors.text)}
-                errorMessage={state.errors.text}
-                infoText={`${state.text.value.length}/${state.text.maxLength} tekens`}
-                name="text"
-                onChange={onTextChange}
-                required={state.text.required}
-                rows={state.text.rows}
-                value={state.text.value || state.text.defaultValue}
-              />
+        <AddNoteWrapper>
+          <QuestionLabel>
+            <strong>{state.text.label}</strong>
+            {!state.text.required && <span>&nbsp;(niet verplicht)</span>}
+            {state.text.required &&
+              state.check.checked &&
+              state.flags.hasEmail && (
+                <Paragraph light>{state.text.subtitle}</Paragraph>
+              )}
+          </QuestionLabel>
+          <ErrorWrapper invalid={Boolean(state.errors.text)}>
+            <div role="status">
+              {state.errors.text && (
+                <ErrorMessage
+                  id="textareaErrorMessage"
+                  message={state.errors.text}
+                />
+              )}
             </div>
 
-            <div>
-              <StyledButton
-                data-testid="statusFormSubmitButton"
-                type="submit"
-                variant="secondary"
-                disabled={disableSubmit}
+            <StandardTextsButton
+              data-testid="standardTextButton"
+              variant="primaryInverted"
+              onClick={openStandardTextModal}
+            >
+              <div>{`Standaardtekst (${defaultTextTemplatesLength()})`}</div>
+            </StandardTextsButton>
+            {modalStandardTextIsOpen && (
+              <Modal
+                data-testid="standardTextModal"
+                open
+                onClose={closeStandardTextModal}
+                title="Standard texts"
               >
-                Opslaan
-              </StyledButton>
+                <DefaultTexts
+                  defaultTexts={defaultTexts}
+                  onHandleUseDefaultText={useDefaultText}
+                  status={state.status.key}
+                  onClose={closeStandardTextModal}
+                />
+              </Modal>
+            )}
 
-              <StyledButton
-                data-testid="statusFormCancelButton"
-                variant="tertiary"
-                onClick={close}
-              >
-                Annuleer
-              </StyledButton>
-            </div>
-          </FormArea>
-          <TextsArea>
-            <DefaultTexts
-              defaultTexts={defaultTexts}
-              onHandleUseDefaultText={setDefaultText}
-              status={state.status.key}
+            <AddNote
+              data-testid="text"
+              isStandalone={false}
+              label={''}
```

Comment:
Shouldn't this field have a label?

Reply:
The label is placed above the button that can open the modal to select a standard text. If left empty, the default label is shown and that is not what we want.